woocommerce/woocommerce-ios
# Spike: Aztec for Product description

This PR is a demo/prototype for editing Product description with Aztec, with basic styles and a few pending tasks.

## Pending tasks
- Refactoring: potentially separate the action handling code from the giant view controller - any ideas on breaking the giant view controller are super welcome!
- Use xib for the view controller
- Display the image instead of a placeholder icon / more advanced media support
- Update Link Settings with nicer UI
- Style polishes

## Testing

- Launch the app with a debug build
- Go to "Products" tab
- Tap on a Product
- Tap on the Product title --> it should navigate to the Product description editor

## Example screenshots

Empty state | Visual mode with various HTML elements | HTML mode
-- | -- | --
![Simulator Screen Shot - iPhone SE - 2019-10-24 at 12 26 23](https://user-images.githubusercontent.com/1945542/67453549-968f3280-f659-11e9-9617-f78e8358209e.png) | ![Simulator Screen Shot - iPhone SE - 2019-10-24 at 12 25 13](https://user-images.githubusercontent.com/1945542/67453561-9f800400-f659-11e9-929d-1a20e728e94f.png) | ![Simulator Screen Shot - iPhone SE - 2019-10-24 at 12 25 42](https://user-images.githubusercontent.com/1945542/67453562-9f800400-f659-11e9-8344-97d47b7336a0.png)

Link Settings | content with links | Toolbar options
-- | -- | --
![Simulator Screen Shot - iPhone SE - 2019-10-24 at 12 28 45](https://user-images.githubusercontent.com/1945542/67453714-23d28700-f65a-11e9-8360-07c5de6371a2.png) | ![Simulator Screen Shot - iPhone SE - 2019-10-24 at 12 30 54](https://user-images.githubusercontent.com/1945542/67453730-2df48580-f65a-11e9-8ae5-ea05d9564ece.png) | ![Simulator Screen Shot - iPhone SE - 2019-10-24 at 12 28 16](https://user-images.githubusercontent.com/1945542/67453729-2df48580-f65a-11e9-8f1b-64ea36e54b5b.png)

Update release notes:
